### ISSUE TITLE
Critical Friendly Zone (CFZ), Circular Symbol Icon Is 'Blue Kill Box' Icon

### ISSUE THREAD

**Issue by david-melendez** [missioncommand/old-mil-sym-java#47]
When generating a symbol icon for the CFZ, Circular symbol, the icon that is generated is for the Blue Kill Box.  Here is the code I used to find the bug:

```
String basepath = "C:\\testdata\\";
        
// Set B standard
RendererSettings s = RendererSettings.getInstance();
s.setSymbologyStandard(0);
        
String id = "GFFPAZFC------X";  // B standard - Critical Friendly Zone, Circular
        
BufferedImage b = r.getMilStdSymbolasIcon(id, 60, Boolean.FALSE);
        
File outFile = new File(basepath + id.replace('*','_') + "icon.png");
ImageIO.write(b, "png", outFile);
```

Here is the image that is generated:

![gffpazfc------xicon](https://cloud.githubusercontent.com/assets/22623059/22609629/8e7b3012-ea27-11e6-9f05-b60686599d25.png)

**Comment by michael-spinelli** [comment]
That is a bug, but at the same time, I don't see a CFZ circular in the USAS or MilStd2525B or C.  I think we may have implemented that by mistake.  Deutch, am I wrong?

**Comment by mdeutch3** [comment]
GFFPAZFC------X does render as a multipoint symbol for B and for C.

**Comment by david-melendez** [comment]
This appears only in the B-Standard (page 506).

 

 

David Melendez

 <mailto:david.melendez@nova-tech-solutions.com> david.melendez@nova-tech-solutions.com

Software Engineer

Nova Tech Solutions

**Comment by mdeutch3** [comment]
Yes, I don't see it in Rev C which only has CFZ irregular and rectangular apparently.

**Comment by michael-spinelli** [comment]
I must've missed it in B. I'll take another look on Monday. Either way, I'll fix the icon issue.

**Comment by michael-spinelli** [comment]
fixed in release v0.1.19
